app/src/lib/shells/win32.ts

```diff
@@ -114,52 +114,35 @@ export async function getAvailableShells(): Promise<
   return shells
 }
 
-function addErrorTracing(context: string, cp: ChildProcess) {
-  cp.stderr.on('data', chunk => {
-    const text = chunk instanceof Buffer ? chunk.toString() : chunk
-    log.debug(`[${context}] stderr: '${text}'`)
-  })
-
-  cp.on('exit', code => {
-    if (code !== 0) {
-      log.debug(`[${context}] exit code: ${code}`)
-    }
-  })
-}
-
-export async function launch(
+export function launch(
```

Comment:
This can also be tidied up with a switch.

Reply:
fe98607d9660958ea4460ffc844d5f97a40740c4